packages/client/src/index.ts

```diff
@@ -12,61 +11,60 @@ export const extractBearerToken = (authorization: string): string => {
     typeof authorization !== 'string' ||
     (!authorization.startsWith('Bearer ') && !authorization.startsWith('bearer '))
   ) {
-    return null;
+    throw new Error('Fail to extract bearer token');
   }
 
   const token = authorization.slice(7);
   return token;
 };
 
 export const ensureBasicOptions = (options?: ConfigParameters): ConfigParameters => {
-  const { clientId, discoveryUrl, redirectUris } = options || {};
-  if (typeof discoveryUrl !== 'string' || !discoveryUrl.startsWith('http')) {
-    throw new Error('Invalid discoveryUrl');
+  const { clientId, logtoUrl } = options || {};
+  if (typeof logtoUrl !== 'string' || !logtoUrl.startsWith('http')) {
+    throw new Error('Invalid logtoUrl');
```

Comment:
1. "ensure" sounds weird in general. maybe "validateSomething"
2. can we align the usage of "options" "config" and "parameters"? the naming is confusing now. `Options` is good to go
3. went through the code i think there's no need to validate the config since we're writing TypeScript. we should always treat the input valid unless needed (like post body in APIs). if we have to validate something, we can just leverage `zod` like logto-io/core.